docusaurus/docs/dev-docs/plugins/development/create-a-plugin.md

```diff
@@ -150,6 +150,76 @@ export default (config, webpack) => {
 };
 ```
 
+:::caution
+When developing your plugin locally (using @strapi/sdk-plugin), your configuration in config/plugins.js might look like this:
+
+```js
+myplugin: {
+    enabled: true,
+    resolve: `./src/plugins/local-plugin`,
+  },
+```
+
+However, this setup can sometimes lead to errors such as:
```

Comment:
I would say something like "since the sdk-plugin is primarly designed to develop plugin not locally, the configuration needs to be adjusted manually for local plugins"